src/k8s-frameworkcontroller/deploy/k8s-frameworkcontroller.json.template

```diff
@@ -0,0 +1,37 @@
+{
+  "apiVersion": "apps/v1",
+  "kind": "StatefulSet",
+  "metadata": {
+    "name": "k8s-frameworkcontroller-sts"
+  },
+  "spec": {
+    "serviceName": "k8s-frameworkcontroller",
+    "selector": {
+      "matchLabels": {
+        "app": "k8s-frameworkcontroller"
+      }
+    },
+    "replicas": 1,
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "k8s-frameworkcontroller"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "k8s-frameworkcontroller",
+            "image": "frameworkcontroller/frameworkcontroller",
```

Comment:
explicitly specify the latest fc version

Reply:
ok

Reply:
updated